src/account/Profile.tsx

```diff
@@ -110,7 +110,7 @@ function Profile({ navigation, route }: Props) {
         </View>
         <View style={styles.disclaimerContainer}>
           <InLineNotification
-            severity={Severity.Informational}
+            variant={Variant.Info}
```

Comment:
```suggestion
            variant={NotificationVariant.Info}
```
maybe? i'm not sure, i wonder if the generic names are confusing. we haven't been terribly consistent with this but the old "Severity" name bothered me. for the button enums, we prefix with "Btn" so it'd be nice to do something similar here?